Support Go v1.11.2 in CI

We need to fix the `gofmt` errors. In centos ci with go version 1.11.2 there's some gofmt errors, that are preventing the integration test from running.
```
07:13:16 + make prerelease synopsis_docs link_check_docs
07:13:16 rm -rf /home/minishift_ci/gopath/pkg/linux_amd64/github.com/minishift
07:13:16 rm -rf out
07:13:16 rm -rf release
07:13:16 rm -f  docs/source/_tmp/*.md
07:13:16 cmd/minishift/cmd/config/config_test.go
07:13:16 cmd/minishift/cmd/config/view_test.go
07:13:16 pkg/minishift/docker/image/oci_image_handler.go
07:13:17 pkg/util/crypto.go
07:13:17 make: *** [fmtcheck] Error 1
```
-----------------------------------
```
07:13:14 + go version
07:13:14 go version go1.11.2 linux/amd64
```

centos ci console: https://ci.centos.org/job/minishift-pr/5484/console

--- a/.circleci/config.yml
+++ b/.circleci/config.yml
@@ -7,7 +7,7 @@ jobs:
     macos:
       xcode: "9.0"
     environment:
-        GOVERSION: 1.10.2
+        GOVERSION: 1.11.2
         XCODE_SCHEME: symbolicator
         XCODE_WORKSPACE: symbolicator
 

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,7 @@
 language: go
 
 go:
-- 1.10.2
+- 1.11.2
 
 install:
 - mkdir -p /home/travis/gopath/bin

--- a/appveyor.yml
+++ b/appveyor.yml
@@ -6,7 +6,7 @@ clone_folder: C:\gopath\src\github.com\minishift\minishift
 
 environment:
   GOPATH: C:\gopath
-  GOVERSION: 1.10.2
+  GOVERSION: 1.11.2
 
 platform:
 - x64

--- a/cmd/minishift/cmd/config/config_test.go
+++ b/cmd/minishift/cmd/config/config_test.go
@@ -50,11 +50,11 @@ var configTestCases = []configTestCase{
     "vm-driver": "kvm"
 }`,
 		config: map[string]interface{}{
-			"iso-url":   "http://foo.bar/minishift-centos.iso",
-			"vm-driver": "kvm",
-			"cpus":      4,
-			"disk-size": "20g",
-			"v":         5,
+			"iso-url":                   "http://foo.bar/minishift-centos.iso",
+			"vm-driver":                 "kvm",
+			"cpus":                      4,
+			"disk-size":                 "20g",
+			"v":                         5,
 			"show-libmachine-logs":      true,
 			"log_dir":                   "/etc/hosts",
 			"ReminderWaitPeriodInHours": 99,

--- a/cmd/minishift/cmd/config/view_test.go
+++ b/cmd/minishift/cmd/config/view_test.go
@@ -30,11 +30,11 @@ type templateExpectedOutputTest struct {
 
 var (
 	configTest = map[string]interface{}{
-		"iso-url":   "http://foo.bar/minishift-centos.iso",
-		"vm-driver": "kvm",
-		"cpus":      4,
-		"disk-size": "20g",
-		"v":         5,
+		"iso-url":                   "http://foo.bar/minishift-centos.iso",
+		"vm-driver":                 "kvm",
+		"cpus":                      4,
+		"disk-size":                 "20g",
+		"v":                         5,
 		"show-libmachine-logs":      true,
 		"log_dir":                   "/etc/hosts",
 		"ReminderWaitPeriodInHours": 99,

--- a/pkg/minishift/docker/image/oci_image_handler.go
+++ b/pkg/minishift/docker/image/oci_image_handler.go
@@ -435,10 +435,10 @@ func (handler *OciImageHandler) getSystemContext(cacheDir string) *types.SystemC
 		DockerDaemonHost:                  handler.dockerClientSettings.DockerHost,
 		DockerDaemonCertPath:              handler.dockerClientSettings.DockerCertPath,
 		DockerDaemonInsecureSkipTLSVerify: !handler.dockerClientSettings.DockerTLSVerify,
-		OSChoice:                    "linux",
-		ArchitectureChoice:          "amd64",
-		OCIAcceptUncompressedLayers: true,
-		OCISharedBlobDirPath:        filepath.Join(cacheDir, "blobs"),
+		OSChoice:                          "linux",
+		ArchitectureChoice:                "amd64",
+		OCIAcceptUncompressedLayers:       true,
+		OCISharedBlobDirPath:              filepath.Join(cacheDir, "blobs"),
 	}
 }
 

--- a/pkg/util/crypto.go
+++ b/pkg/util/crypto.go
@@ -62,7 +62,7 @@ func GenerateSelfSignedCert(certPath, keyPath string, ips []net.IP, alternateDNS
 		KeyUsage:              x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
 		ExtKeyUsage:           []x509.ExtKeyUsage{x509.ExtKeyUsageServerAuth, x509.ExtKeyUsageClientAuth},
 		BasicConstraintsValid: true,
-		IsCA: true,
+		IsCA:                  true,
 	}
 
 	template.IPAddresses = append(template.IPAddresses, ips...)

--- a/pkg/util/os/atexit/atexit.go
+++ b/pkg/util/os/atexit/atexit.go
@@ -74,7 +74,7 @@ func runHandler(exitHandler func(code int) bool, code int) bool {
 	defer func() {
 		err := recover()
 		if err != nil {
-			glog.Exitf("Error running exit handler:", err)
+			glog.Exitf("Error running exit handler: %v", err)
 		}
 	}()
 

